docs/index.md

```diff
@@ -6,37 +6,47 @@ Project documentation with&nbsp;Markdown.
 
 ## Overview
 
-MkDocs is a **fast**, **simple** and **downright gorgeous** static site generator that's geared towards building project documentation. Documentation source files are written in Markdown, and configured with a single YAML configuration file.
+MkDocs is a **fast**, **simple** and **downright gorgeous** static site
+generator that's geared towards building project documentation. Documentation
+source files are written in Markdown, and configured with a single YAML
+configuration file.
 
----
-
-**MkDocs is currently still in development.**
+!!! warning
```

Comment:
The linter complained that bold was being used instead of a header. While understandable, this is **not** a header. However, it is a warning to I changed it to an admonition (which I suspect was not an option when this was originally authored).